server/controllers/admin.ts

```diff
@@ -187,7 +187,7 @@ export const addBatch = async function(req, res) {
         // expRound is randomized. It's [numRounds] or [numRounds - 1]
         const min = newBatch.numRounds - 1;
         const roundNumber = Math.floor(Math.random() + 0.5) + min; //random int from min to max
-
+        console.log("Random Experiment Round: " + roundNumber);
```

Comment:
Do we need to log this? Isn't it stored in the DB? (I don't mind either way, I'm just always conscious that putting more in the logs usually makes it harder to find what we want to see there)

Reply:
I'll remove!